src/state/routing/types.ts

```diff
@@ -187,6 +194,23 @@ export class ClassicTrade extends Trade<Currency, Currency, TradeType> {
     this.requestId = requestId
     this.quoteMethod = quoteMethod
     this.approveInfo = approveInfo
+    this.inputTax = inputTax
+    this.outputTax = outputTax
+  }
+
+  public get totalTaxRate(): Percent {
+    return this.inputTax.add(this.outputTax)
+  }
+
+  public get postTaxOutputAmount() {
+    return this.outputAmount.multiply(new Fraction(ONE).subtract(this.totalTaxRate))
```

Comment:
can you also add a comment about why we're doing the semi-jank total tax rate 

something like:
- ideally we should amend the `inputAmount` based on the input tax but its a bit tricky because V2Trade reconstructs the total `inputAmount` based on the swap routes
- TODO: Amend V2Trade objects in the v2-sdk to have a separate field for post-input tax routes